cookbooks/swift/recipes/source.rb

```diff
@@ -109,7 +109,7 @@
 
 execute "python-swift-install" do
   cwd "#{node['source_root']}/swift"
-  command "pip install -e .[kmip_keymaster] && pip install -r test-requirements.txt"
+  command "pip install -c py2-constraints.txt -e .[kmip_keymaster] && pip install -c py2-constraints.txt -r test-requirements.txt"
```

Comment:
I'd strongly prefer that we make this conditional on `USE_PYTHON3` not being true.